fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -1363,8 +1363,7 @@ private void computeFragmentHosts() throws Exception {
             if (params.instanceExecParams.isEmpty()) {
                 Reference<Long> backendIdRef = new Reference<Long>();
                 TNetworkAddress execHostport;
-                if (ConnectContext.get() != null
-                        && !ConnectContext.get().isResourceTagsSet()
+                if (ConnectContext.get() != null && ConnectContext.get().isResourceTagsSet()
```

Comment:
Why change here?

Reply:
the following the comment said，if resource tag is set，so I think it is a bug here